### ISSUE TITLE
react 17?

### ISSUE THREAD

**Issue by adenta** [Hyperting/hypertheme-editor#17]
When I tried to install this to my repo, I got a dependency error:

```
➜  criya git:(hypertheme) npm i @hypertheme-editor/chakra-ui
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR!
npm ERR! While resolving: app@undefined
npm ERR! Found: react@17.0.2
npm ERR! node_modules/react
npm ERR!   react@"^17.0.2" from the root project
npm ERR!   peer react@">=16.8.6" from @chakra-ui/react@1.7.4
npm ERR!   node_modules/@chakra-ui/react
npm ERR!     @chakra-ui/react@"^1.7.3" from the root project
npm ERR!     peer @chakra-ui/react@">= 1.0.0" from @hypertheme-editor/chakra-ui@0.1.2
npm ERR!     node_modules/@hypertheme-editor/chakra-ui
npm ERR!       @hypertheme-editor/chakra-ui@"*" from the root project
npm ERR!   5 more (@emotion/react, @emotion/styled, framer-motion, ...)
npm ERR!
npm ERR! Could not resolve dependency:
npm ERR! peer react@"^16.8.0" from @hypertheme-editor/chakra-ui@0.1.2
npm ERR! node_modules/@hypertheme-editor/chakra-ui
npm ERR!   @hypertheme-editor/chakra-ui@"*" from the root project
npm ERR!
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
npm ERR!
npm ERR! See /Users/andre/.npm/eresolve-report.txt for a full report.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/andre/.npm/_logs/2022-01-09T17_17_44_423Z-debug-0.log
```

is this project abandoned?

**Comment by it-nalon** [comment]
Hi, the project is not abandoned at all.
We're working on the 0.2.x version and right now we're focusing mostly on pro features (on a private repo).

I see the problem here, I've published a `0.1.3-rc.0` version.

Could you try to install this version number? If it works we'll publish the fix as `0.1.3`.

Many thanks for opening the issue, much appreciated!

**Comment by adenta** [comment]
confirming the install work...testing repo now

**Comment by adenta** [comment]
it works!

**Comment by it-nalon** [comment]
Hi, we just published the 0.1.3 version, so you can switch: `0.1.3-rc.0` to: `0.1.3`.
I'm closing the issue, feel free the open a new one for any issue.

Thank you very much for your help!
